OpenConsole.sln

```diff
@@ -239,7 +239,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{89CDCC
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Types.Unit.Tests", "src\types\ut_types\Types.Unit.Tests.vcxproj", "{34DE34D3-1CD6-4EE3-8BD9-A26B5B27EC73}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "PublicTerminalCore", "src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj", "{84848BFA-931D-42CE-9ADF-01EE54DE7890}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Microsoft.Terminal.Control.ABI", "src\cascadia\TerminalControl\dll_abi\Microsoft.Terminal.Control.ABI.vcxproj", "{84848BFA-931D-42CE-9ADF-01EE54DE7890}"

```

Comment:
I'm not entirely sold on the naming... Isn't WinRT also an ABI?

Oh btw... Just wondering... How large would be a `Microsoft.Terminal.Control.dll` that contains both, the WinRT and C API? (= 1 binary = simpler shipping?) Did you check that / have an estimate?

Reply:
I don't give a single `****` about the filename, I just wanted something to put on disk.

It's 2MB. It would be simpler, and it is how I have been prototyping it.

However: It costs the 150MM+ Terminal users in package size, and it costs the multiple millions of Visual Studio users in on-disk install size. It might be worth a minor amount of complication to save that much disk and transit in aggregate.

Oh, and we can use different linker flags for different versions - use onecoreuap for one, delayload for another, etc.

Reply:
As it stands with HwndTerminal being 100% divergent on the Core and Interactivity layer:

File | Size
-----|------
Terminal.Control.ABI.dll | 637952
Terminal.Control.dll | 2077696
Terminal.Control.dll plus ABI exports | 2100224

I don't have numbers for HwndTerminal + Interactivity platform, but at last check the DLL was like 1200k?

Now that I see the change in Terminal.Control, I no longer think the separate DLL thing is worth doing ;P

the WPF control can safely get larger. VS already isn't small :P